windows/setup/handle_vmci_known_issue.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# ESXi version 8.0.1 to 8.0.3 with build lower than ESXi 8.0 Update 3c
+# Build 24414501, there is known issue on Windows VMs,
+# that guest OS hang after reverting to 'BaseSnapshot' when VM VBS is enabled.
+#
+- name: "Get VM VBS enable status"
+  include_tasks: ../../common/vm_get_vbs_status.yml
+
+- name: "VM VBS is enabled"
+  when: vm_vbs_enabled
+  block:
+    - name: "Get VMware Tools status"
+      include_tasks: ../../common/vm_get_vmtools_status.yml
+
+    - name: "Save VMware Tools status"
+      ansible.builtin.set_fact:
+        vmtools_is_installed_before: "{{ vmtools_is_installed }}"
+        vmtools_is_running_before: "{{ vmtools_is_running }}"
+        get_vmtools_info: {}
+
+    # VMware Tools status will become not running
+    - name: "Refresh VMware Tools running status"
+      when:
+        - vmtools_is_installed_before
+        - vmtools_is_running_before
+      block:
+        - name: "Waiting for VMware Tools is not running"
+          include_tasks: ../../common/vm_wait_vmtools_status.yml
+          vars:
+            vm_wait_vmtools_running: false
+            vm_wait_vmtools_ignore_error: true
+        - name: "Save VMware Tools running status"
+          ansible.builtin.set_fact:
+            vmtools_is_running_after: false
+          when:
+            - get_vmtools_info.vmtools_info.vm_tools_running_status is defined
+            - get_vmtools_info.vmtools_info.vm_tools_running_status == 'guestToolsNotRunning'
+
+    - name: "Meet known issue conditions"
+      ansible.builtin.set_fact:
+        vmci_known_issue: "{{ (vmtools_is_installed_before and not vmtools_is_running_before) or (not vmtools_is_running_after | default(true)) }}"
```

Comment:
`vmtools_is_running_after | default(true)` The default value is kind of misleading literally. Afterall, tools could be not installed.  I suggest setting vmci_known_issue in tasks of line 18 and line 36 separately: 
In line 18 task:
```
vmci_known_issue: "{{ vmtools_is_installed_before and not vmtools_is_running_before }}"
```
In line 36 task:
```
vmci_known_issue: true
```